tests/Client/UpdaterTest.php

```diff
@@ -536,7 +524,7 @@ public function providerVerifiedDelegatedDownload(): array
      *
      * § 5.6.7.1
      */
-    public function testMaximumRoles(): void
+    public function testRoleDownloadsAreLimited(): void
```

Comment:
When I renamed test methods like this, it was to make them clearer in testdox output (which also tends to make them clearer in code as well).